tiledb/sm/cpp_api/query_condition.h

```diff
@@ -210,6 +210,31 @@ class QueryCondition {
     return QueryCondition(ctx_, combined_qc);
   }
 
+  /**
+   * Return a query condition representing a negation of
```

Comment:
NIT: this comment could be made to be 2-3 lines.

Reply:
Done.